ocs_ci/templates/app-pods/awscli_multiarch.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
 spec:
   containers:
     - name: awscli
-      image: quay.io/multiarch-origin-e2e/aws-cli:latest
-      # Override the default `aws` entrypoint in order to 
+      image: quay.io/ocsci/s3-cli-with-test-objects-multiarch:latest
```

Comment:
I do not know whether the `latest` tag is supported since I only ever pushed `1.0`